src/popup/utils/utils.ts

```diff
@@ -600,7 +600,7 @@ export async function readValueFromClipboard(): Promise<string | undefined> {
   }
   let text = '';
 
-  if (IS_CORDOVA) {
+  if (IS_IONIC) {
```

Comment:
Maybe instead of using direct framework names like "cordova" or "ionic" we could have something more generic? Like `IS_MOBILE_APP`?